### PR TITLE
navi 2.5.0

### PR DESCRIPTION
**Pull request by denisidoro**
Changelog: https://github.com/denisidoro/navi/releases/tag/v2.5.0

### CODE REVIEW

File: Formula/navi.rb [hunk 1]
```diff
@@ -13,6 +13,7 @@ class Navi < Formula
 
   depends_on "rust" => :build
   depends_on "fzf"
+  depends_on "git"
```
Comment (Bo98) [inline Formula/navi.rb:16]: Is system git not sufficient?

Comment (denisidoro) [reply Formula/navi.rb:16]: Sorry, what does `system git` means exactly?
`navi` now shell outs and calls `git ...`
I just want to make sure that `git` is available when running `navi`
If `git` is guaranteed to be available even without this line, then this line can be removed
Is that the case?

Comment (Bo98) [reply Formula/navi.rb:16]: You can assume git is available as Homebrew uses git itself to update, unless you specifically require a recent version of git.

**Comment by denisidoro** [comment]
Closing without merging because I'll probably make the changes for the 2.5.1 release

**Pull request closed without merging**